maths/reverse_a_number.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import print_function
+
+
+def reverse_num(num: str) -> str:
```

Comment:
It's more accurate to say that this function reverses a _string,_ not just a number, because you're taking strings, not ints, as input. Please provide an implementation that uses numerical operations to reverse a number instead, as doing this with strings is basically cheating IMO.